src/containers/MyNdla/Arena/ArenaAdminPage.tsx

```diff
@@ -9,24 +9,44 @@
 import { useContext } from "react";
 import { useTranslation } from "react-i18next";
 import { Navigate } from "react-router-dom";
-import styled from "@emotion/styled";
-import { spacing } from "@ndla/core";
 import { UserLine, AlertLine } from "@ndla/icons/common";
+import { Heading } from "@ndla/primitives";
+import { css } from "@ndla/styled-system/css";
+import { styled } from "@ndla/styled-system/jsx";
 import { HelmetWithTracker } from "@ndla/tracker";
-import { Heading, Text } from "@ndla/typography";
+import { Text } from "@ndla/typography";
```

Comment:
Feil Text? Eller blir denne tatt i en annen PR?

Reply:
Egen PR https://github.com/NDLANO/ndla-frontend/pull/2008 tar omskrivinga for Arena-sidene, denne PRen gjelder i hovedsak kun brukere / flagg liste/tabellene (for å unngå diger PR).